stuf_zds_payments/client.py

```diff
@@ -0,0 +1,73 @@
+from functools import partial
+
+from django.utils import timezone
+
+from openforms.logging import logevent
+from stuf.constants import EndpointType
+from stuf.models import StufService
+from stuf.service_client_factory import ServiceClientFactory, get_client_init_kwargs
+from stuf.stuf_zds.client import (
+    Client as _Client,
+    NoServiceConfigured,
+    PaymentStatus,
+    ZaakOptions,
+    fmt_soap_date,
+)
+from stuf.stuf_zds.models import StufZDSConfig
+
+
+class ZaakOptions(ZaakOptions):
+    payment_status_update_xml: dict
```

Comment:
shouldn't this be a string?

```suggestion
    payment_status_update_xml: str
```